mungegithub/options/options.go

```diff
@@ -456,12 +457,29 @@ func toString(optType optionType, val interface{}) string {
 	}
 }
 
+func (o *Options) keysSortedAndWidth() ([]string, int) {
+	keys := make([]string, len(o.options))
```

Comment:
Because it's shorter and performance doesn't matter for functions called single-digit numbers of times each day. ;)

If you really want to do it, use `make([]string, 0, len(o.options))`-- that makes an empty string slice with capacity of o.options, so it won't cause any allocations as you append to it.